README.md

```diff
@@ -29,48 +29,12 @@ This SDK supports:
 * UIKit
 * SwiftUI
 
-## Access Token
+## Client ID
 
-The PayPal SDK uses access tokens for authentication.
+The PayPal SDK uses client ID for authentication.
```

Comment:
```suggestion
The PayPal SDK uses a client ID for authentication.
```

Reply:
Addressed in 44250f07b9791b6cac9cca79803900a17b1d10ee